compiler/src/dotty/tools/dotc/parsing/Scanners.scala

```diff
@@ -1359,7 +1359,8 @@ object Scanners {
 
   case class InBraces(var width: IndentWidth | Null, outer: Region)
```

Comment:
So how is passing null different from passing IndentWidth.Zero here ? Some documentation might be helpful.